client/app/cloud/project/compute/infrastructure/virtualMachine/addEdit/translations/Messages_fr_FR.xml

```diff
@@ -58,7 +58,7 @@
    <translation id="cpci_vm_flavor_category_s1-2" qtlid="377389">S1-2</translation>
    <translation id="cpci_vm_flavor_category_s1-4" qtlid="377402">S1-4</translation>
    <translation id="cpci_vm_flavor_category_s1-8" qtlid="377415">S1-8</translation>
-
+   <translation id="cpcivm_addedit_flavor_has_old_flavors">Anciens modèles</translation>
```

Comment:
Detail but you could've skipped the "has" in there.. the string is more meaningful without it :)